plugin/src/main/java/org/opensearch/ml/processor/MLInferenceIngestProcessor.java

```diff
@@ -49,6 +54,7 @@ public class MLInferenceIngestProcessor extends AbstractProcessor implements Mod
     // prediction outcomes, return the whole prediction outcome by skipping filtering
     public static final String IGNORE_MISSING = "ignore_missing";
     // At default, ml inference processor allows maximum 10 prediction tasks running in parallel
+    // it can be overwritten using maxPredictionTask when creating processor
```

Comment:
Use snake case for processor configuration keys ? for example `input_map` , we don't use camel case `inputMap`. Let's keep consistent

Reply:
Most of the processors' config are using snake key, for example 
 
https://github.com/opensearch-project/OpenSearch/blob/fc81a906da88d99c7ea09ffd9fec92ff81925cf6/modules/ingest-common/src/main/java/org/opensearch/ingest/common/KeyValueProcessor.java#L289-L293


https://github.com/opensearch-project/OpenSearch/blob/fc81a906da88d99c7ea09ffd9fec92ff81925cf6/modules/search-pipeline-common/src/main/java/org/opensearch/search/pipeline/common/FilterQueryRequestProcessor.java#L40

Reply:
I see https://github.com/opensearch-project/ml-commons/blob/6f02ebc403c7b17fed08386be4fc74641b2aabba/plugin/src/main/java/org/opensearch/ml/processor/InferenceProcessorAttributes.java#L63
```
public static final String MAX_PREDICTION_TASKS = "max_prediction_tasks";
```

Not a big concern, but to avoid confusion, please update `maxPredictionTask` in this comment to `max_prediction_tasks`.